package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "android": "node ./build/scons cleanbuild android",
     "build": "node ./build/scons build",
-    "build:local": "node ./build/scons build && node ./build/scons package --skip-zip && ./build/scons install",
+    "build:local": "node ./build/scons build && node ./build/scons package --skip-zip && node ./build/scons install",
```

Comment:
this one shouldn't be in this PR. Always start from the normal master branch

Reply:
Initialy I wanted to make two PR, but I have to correct this one if I wanted that compilation to work.

I dont understand how people are supposed to compile on Windows as it is buggy, anyway no idea on how to split the two commits now in two separated PR ? (I mean without too much complexity)

Reply:
I would remove the fork on your machine, clone it again so you have a fresh folder. Then create a branch `git checkout -b fixModuleBuild` and copy the code from here into that branch. Commit and push. Then go the master again (so its still without any changes) and create a second branch `git checkout -b secondFeature` and copy the code from the other PR into that, commit, push. Then you always have a clean master without any changes and two separate branches for your PRs

Reply:
That was my first idea :), I was just looking for something less tedious, but if it is the good workflow for Titanium, I will do that